backend/molgenis-emx2-io/src/main/java/org/molgenis/emx2/io/emx2/Emx2.java

```diff
@@ -279,4 +288,28 @@ public static List<Row> toRowList(SchemaMetadata schema) {
     }
     return result;
   }
+
+  public static String sanitize(String value) {
+    // main purpose is to remove spaces because not allowed in names, and also to make nice
+    // camelCase or PascalCase (we don't touch first character to not destroy existing names)
+    if (value != null) {
+      String[] words = value.split("\\s+");
+      StringBuilder result = new StringBuilder();
+      for (int i = 0; i < words.length; i++) {
+        if (i == 0) {
+          result.append(words[i]);
+        } else {
+          // all other words have first letter character case
+          result.append(words[i].substring(0, 1).toUpperCase());
+
+          if (words[i].length() > 1) {
+            result.append(words[i].substring(1));
+          }
+        }
+      }
+      return result.toString().trim();
+    } else {
+      return null;
+    }
+  }
```

Comment:
There should be an exhaustive unit test for this function.

Reply:
this is a temporary method that I want to remove once all existing models in 'data' have been updated to adhere to the new rules. This will need to be done in seperate request.